packages/compass-web/package.json

```diff
@@ -84,6 +84,7 @@
     "@mongodb-js/compass-welcome": "^0.40.0",
     "@mongodb-js/compass-workspaces": "^0.23.0",
     "@mongodb-js/connection-storage": "^0.21.0",
+    "@mongodb-js/devtools-proxy-support": "^0.3.9",
```

Comment:
I'm pretty sure that this missing is why the compass-web e2e are not running right now. I'll do a follow-up to make sure that if building compass-web failed in e2e, we are not just silently finishing the tests with success

Reply:
oof. Wondering how that happens. (task succeeding even though it failed)

Reply:
Yeah, I'm also confused by that, at a glance it really shouldn't... 😕